### PR TITLE
Hello World 1-task

### PR DESCRIPTION
**Pull request by utkir1212**
Men pechat qildim

### CODE REVIEW

File: My Test Project/src/main/java/Main.java [hunk 1]
```diff
@@ -2,4 +2,8 @@
  * Created by UTKIRI on 29.07.2018.
  */
 public class Main {
+    public static void main(String[] args) {
```
Comment (Sunedge) [inline My Test Project/src/main/java/Main.java]: Why Do we need Main class

Comment (utkir1212) [reply My Test Project/src/main/java/Main.java]: ichim qisdi shuni uchun

Comment (Sunedge) [inline My Test Project/src/main/java/Main.java]: Asa ina sanga

**Comment by Sunedge** [comment]
tesge

**Pull request merged**